interface/drugs/add_edit_drug.php

```diff
@@ -374,7 +374,6 @@ function sel_related() {
 <h3 class="ml-1"><?php echo text($title);?></h3>
 <form class="form" method='post' name='theform' action='add_edit_drug.php?drug=<?php echo attr_url($drug_id); ?>'>
 <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
-<center>
 
 <table class="table table-borderless w-100">
```

Comment:
Hello @tywrenn 

Such as this table form, should I change to the Bootstrap form style?

Reply:
You definitely should since tables are less responsive than the Bootstrap grid classes. It'll take a bit to convert the form yet it's definitely worth it.

Reply:
Ok, I will convert it to the bootstrap form :)